services/api/src/routes/game.ts

```diff
@@ -7,8 +8,14 @@ router.use((_req, res, next) => {
   next();
 });
 
-router.get('/test', async (_req, res) => {
-  res.json({ test: true });
+router.get('/:id', async (req, res) => {
+  const pathUUID = req.params.id;
+  const uuidValidationSchema = Joi.string().guid({
+    version: 'uuidv4',
+    separator: '-',
+  });
+  const authResult = uuidValidationSchema.validate(pathUUID);
```

Comment:
Since you use `async` function, please use `validateAsync` method and `try-catch` blocks